cognitedata/reveal
# fix: display correct 360 image annotation for each revision

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-905

## Description :pencil:


Since annotations are added as children on the visualization box, and the visualization box only exists per entity, not revision, separating annotations between revisions has been slightly non-trivial.

With this change, visiting a revision shows the right annotations for that revision.

## How has this been tested? :mag:



- Start Reveal examples
- Open any 3d-test model (e.g. https://localhost:3000/?project=3d-test&env=greenfield-3d&modelId=4923307033982497&revisionId=3100284276970593)
- Add 360 image set `XOM-HCU1_north_v02`
- Enter correct image entity (shown in image below)
- Alternate between values 0 and 1 in the "Current image revision" field in the UI
- Observe that the revision annotations switch back and forth correctly

![bg0](https://user-images.githubusercontent.com/70905152/235160606-baab1899-ee38-4795-af3e-955ab117b99d.png)



## Test instructions :information_source:




## Checklist :ballot_box_with_check:
